01_install_requirements.sh

```diff
@@ -19,6 +19,26 @@ if [ ! -f /etc/yum.repos.d/epel.repo ] ; then
     fi
 fi
 
+# Install qemu-img-ev on RHEL
```

Comment:
On my system this comes from rdo-qemu-ev, which is enabled via tripleo-repos, do we just need to move the install until after tripleo-repos has been run perhaps?